src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -64,12 +67,37 @@ public ProcessMetrics(ProcessInstrumentationOptions options)
             },
             unit: "By",
             description: "The amount of virtual memory allocated for this process that cannot be shared with other processes.");
+
+        this.MeterInstance.CreateObservableCounter(
+            "process.cpu.time",
+            () =>
+            {
+                if (!this.userProcessorTime.HasValue || !this.privilegedProcessorTime.HasValue)
+                {
+                    this.Snapshot();
+                }
+
+                var userProcessorTimeValue = this.userProcessorTime.Value;
+                var privilegedProcessorTimeValue = this.privilegedProcessorTime.Value;
+                this.userProcessorTime = null;
+                this.privilegedProcessorTime = null;
+
+                return new[]
+                {
+                    new Measurement<double>(userProcessorTimeValue, new KeyValuePair<string, object?>("state", "user")),
+                    new Measurement<double>(privilegedProcessorTimeValue, new KeyValuePair<string, object?>("state", "system")),
+                };
+            },
+            unit: "s",
+            description: "Total CPU time broken down by user state and system state.");
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md#process Could we use same description as the spec?

Reply:
"Total CPU seconds broken down by different states" or "Total CPU seconds broken down by user state and system state."?

Reply:
Hmm... I think it will be clearer if what being the "different states" are specified in the description.

Reply:
unless there is a reason, I'd stick with that the spec doc has.